src/platform/test/functional/apps/discover/group1/_discover.ts

```diff
@@ -103,15 +103,15 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await discover.clickHistogramBar();
         await discover.waitUntilSearchingHasFinished();
         const time = await timePicker.getTimeConfig();
-        expect(time.start).to.be('Sep 21, 2015 @ 12:00:00.000');
-        expect(time.end).to.be('Sep 21, 2015 @ 15:00:00.000');
+        expect(time.start).to.be('Sep 21, 2015 @ 09:00:00.000');
```

Comment:
I had to update these assertions to pass the tests. Here's a screenshot running the tests from `main`:

![es70-discover-test-01-before](https://github.com/user-attachments/assets/418dc693-7542-4a9a-8719-86bc07e17ce1)

Here's screenshot from this branch:

![es70-discover-test-02-after](https://github.com/user-attachments/assets/b892db1f-6333-4aac-a9a0-17a96efedef8)

You can see that the chart has the updated multilayer time axis and the wide right margin is gone. I'm not sure if that justifies the dates in these assertions to change? Maybe someone with more knowledge about these tests can chime in.